docs/algorithms.md

```diff
@@ -84,7 +90,33 @@ To learn more about Guided GradCAM, visit the following resources:
 - [Original paper](https://arxiv.org/abs/1610.02391)
 - [Website](http://gradcam.cloudcv.org/)
 
-Note: this method is currently available in master and will be available in pip / conda releases soon.
+### Feature Ablation
+Feature ablation is a perturbation based approach to computing attribution, involving replacing each input feature with a given baseline / reference value (e.g. 0), and computing the difference in output. Input features can also be grouped and ablated together rather than individually.
```

Comment:
nit: sometimes we say `to computing` sometimes `to compute`. it would be good to be consistent and call it `to compute`.